messages/extend-config-missing.txt

```diff
@@ -0,0 +1,3 @@
+ESLint couldn't find the config "<%- configName %>" to extend from. Please, check the name of the config is correct.
+
+If you still can't figure out the problem, please stop by https://gitter.im/eslint/eslint to chat with the team.
```

Comment:
Nitpick: Not a big fan of `"If you still can't figure out the problem"` - sounds a bit accusatory. Alternative suggestion: `"If you are still having difficulties after checking the above, please stop by https://gitter.im/eslint/eslint to chat with the team."`